packages/forms/src/Form.test.js

```diff
@@ -88,24 +88,28 @@ describe('renderActions', () => {
 				type: 'button',
 				style: 'link',
 				label: 'CANCEL',
+				onClick: () => {},
 			},
 			{
-				type: 'submit',
+				type: 'button',
 				style: 'primary',
 				label: 'VALIDATE',
+				onClick: () => {},
+			},
+			{
+				type: 'submit',
+				style: 'primary',
+				label: 'SUBMIT',
+				disabled: true,
+				onClick: () => {},
 			},
 		];
 
 		// when
 		const wrapper = shallow(<div>{renderActions(actions, noop)}</div>);
 
 		// then
-		expect(wrapper.find(Button)).toHaveLength(2);
-	});
-
-	it('should render a single submit button', () => {
```

Comment:
i think this test should be keeped and adapter, since the default behavior of the tested function is to return and Action if none provided by the user.